assets/wizards/analytics/views/configuration/style.scss

```diff
@@ -0,0 +1,37 @@
+@import '~@wordpress/base-styles/colors';
+
+.newspack__analytics-configuration {
+	th {
+		text-align: left;
+		border-bottom: 1px solid $light-gray-500;
+	}
+	table {
+		width: 100%;
+		margin-bottom: 4em;
+		border-spacing: 0;
+	}
+	tbody {
+		td {
+			padding-top: 4px;
+		}
+		.newspack-checkbox-control {
+			margin: 0;
+		}
+	}
+	&__form {
+		display: flex;
+		align-items: flex-end;
+		.newspack-text-control {
+			width: 100%;
+		}
+		> * {
+			margin: 0 !important;
+			&:not( :last-child ) {
+				margin-right: 20px !important;
+			}
+		}
+		button {
+			flex-basis: 33%;
```

Comment:
This one is odd, because it always looks ok on my machine (without neither `flex-basis` nor `overflow` properties). Can you have a look @dkoo , if this change would still fix the original issue, while not making it reappear on smaller screens?

Reply:
`overflow: visible;` does seem to fix the issue for me on both desktop and mobile, but on mobile, it still looks smooshed:

<img width="377" alt="Screen Shot 2020-07-22 at 9 48 59 AM" src="https://user-images.githubusercontent.com/2230142/88198157-a5af0b00-cc00-11ea-9e57-0fc12abf654b.png">

Normally, I would suggest making a media query to orient these elements vertically at smaller viewport sizes and add some vertical margins between them, but I'll defer to @laurelfulford on whether that's necessary.

Reply:
Does fdcb2dce058ae0aefdaf129cb2f850e6409eac43 help?

Reply:
This looks good to me!